package.json

```diff
@@ -393,6 +397,37 @@
     "configuration": {
       "title": "MongoDB",
       "properties": {
+        "mongodbLanguageServer.maxNumberOfProblems": {
+          "scope": "resource",
+          "type": "number",
+          "default": 100,
+          "description": "Controls the maximum number of problems produced by the server."
+        },
+        "mongodbLanguageServer.trace.server": {
+          "scope": "window",
```

Comment:
I prefer to have the possibility to manage logs level.

1) We can configure other logs verbosity using settings provided by VSCode by default.

<img width="1203" alt="Screenshot 2020-04-15 at 17 32 29" src="https://user-images.githubusercontent.com/16307679/79356545-673dec00-7f3f-11ea-8d63-87c8509645f6.png">

2) Since users of our extension are developers it can help them as well debugging the playground and open issues if our language server has a bug. I have seen requests to other extensions' developers from their users to allow configure logs, so I believe it can be valuable.

Reply:
Cool sounds good.